src/pages/Overview/index.js

```diff
@@ -1,16 +1,642 @@
+    dispatch({
+      type: 'dashboard/updateResults',
+      payload: updatedResult,
+    }).then(() => {
+      this.setState({
+        totalResultData: updatedResult,
+      });
+      this.getSeparatedResults();
+    });
+  };
+
+  retrieveResults = row => {
+    const { dispatch } = this.props;
+    this.markResultSeen(row);
+    dispatch({
+      type: 'global/updateSelectedResults',
+      payload: [row],
+    }).then(() => {
+      dispatch(
+        routerRedux.push({
+          pathname: '/result',
+        })
+      );
+    });
+  };
+
+  navigateToExpiringResult = () => {
+    const { dispatch } = this.props;
+    dispatch(
+      routerRedux.push({
+        pathname: '/expiring-results',
+      })
+    );
+  };
+
   render() {
+    const { newData, unlabeledData } = this.state;
+    const { favoriteResults, seenResults } = this.props;
+    const newDataColumns = [
+      {
+        Header: 'Result',
+        accessor: 'result',
+        Cell: cell => {
+          const row = cell.row.original;
+          return seenResults.includes(row.result) ? (
+            <div>
+              <Button variant="link" isInline style={{ marginBottom: '8px' }}>
+                {cell.value}
+              </Button>
+              <br />
+              <Text component={TextVariants.p} className={styles.subText}>
+                <span className={styles.label}>{row.controller}</span>
+              </Text>
+            </div>
+          ) : (
+            <div>
+              <Button variant="link" isInline style={{ marginBottom: '8px' }}>
+                <b>
+                  <i>{cell.value}</i>
+                </b>
+              </Button>
+              <br />
+              <Text component={TextVariants.p} className={styles.subText}>
+                <span className={styles.label}>{row.controller}</span>
+              </Text>
+            </div>
+          );
+        },
+      },
+      {
+        Header: 'End Time',
+        accessor: 'end',
+        Cell: cell => (
+          <Tooltip content={formatDate('utc', cell.value)}>
+            <span>{formatDate('with time', cell.value)}</span>
+          </Tooltip>
+        ),
+      },
+      {
+        Header: 'Scheduled for deletion on',
+        accessor: 'deletion',
+        Cell: cell => {
+          const remainingDays = getDiffDays(cell.value);
+          return (
+            <div>
+              {remainingDays > 15 ? (
+                <Tooltip content={formatDate('without time', cell.value)}>
+                  <span>{getDiffDate(cell.value)}</span>
+                </Tooltip>
+              ) : (
+                <span>{cell.value}</span>
+              )}
+              <Progress
+                min={0}
+                max={EXPIRATION_LIMIT.val}
+                value={EXPIRATION_LIMIT.val - remainingDays}
+                size={ProgressSize.sm}
+                measureLocation={ProgressMeasureLocation.none}
+              />
```

Comment:
A progress bar? This seems a bit odd, since you seem to be showing the "progress" of the dataset's expiration window relative to the arbitrary maximum allowed, which doesn't really seem all that useful to me.

And using the `EXPIRATION_LIMIT` literal from your mock spreads knowledge of the mock further than necessary through the code. Can't we isolate this in Redux where we can seamlessly transition from built-in mocks to real data when it's available, without having to find all the places that were faked?

I anticipate two different "expiration limits", at the server level (controlled by the server administrator) and at the user level, which would appear as user metadata. (We don't have either yet, unfortunately, although Nikhil's user metadata PR provides the mechanism for the latter just as my dataset metadata provides the basis for your `SEEN` and `SAVED` metadata.)

We should expand the user context in Redux to include a per-user maximum expiration, which can be displayed and (eventually) edited as part of the user profile. This would be limited (in the server) against a server maximum value controlled by the server administrator. (I'd also eventually like to have a server administration console in the dashboard that allows an ADMIN user to interact directly with those, but we're not nearly there yet.)

Anyway, the point is that if you mocked things like this cleanly within a Redux context we won't have to touch so much code to roll out the server feature; and the dashboard code will look more consistent.